SwrveSDK/SDK/Talk/SwrveMessageController.m

```diff
@@ -55,6 +55,7 @@ @interface SwrveMessageController()
 @property (nonatomic, retain) NSString*             apiKey;
 @property (nonatomic, retain) NSArray*              campaigns; // List of campaigns available to the user.
 @property (nonatomic, retain) NSMutableDictionary*  campaignsState; // Serializable state of the campaigns.
+@property (nonatomic, retain) NSLock*               campaignsStateLock;
```

Comment:
Although it says here ios8+
https://developer.apple.com/reference/foundation/nslock